docs/guides/administrator/adding-and-managing-users.md

```diff
@@ -0,0 +1,24 @@
+# Adding and Managing Users
+
+## Introduction
+
+An administrator can provision and manage users in Opstrace.
+Users allow authentication against our web UI and the per-tenant Grafana UI.
+An administrator can then* provide fine-grained permissions for what users can and cannot do.
+
+![users management UI overview](../../assets/users-guide-overview-1.jpg)
+
+\* _This is a roadmap item; all users are currently granted the ADMIN role_
+
+## How to add a user
+
+Adding a user is easy.
+
+![adding a user gif](../../assets/users-guide-add.gif)
+
+## How to remove a user
+
+Removing a user can be done.
+However, as you might expect, **this operation cannot be undone**.
```

Comment:
> **this operation cannot be undone**.

Let's clarify what this means. For example, can't I re-add the same user (as identified by the same email address as before)?

Do we mean to say that any associated relational data will be deleted, too? (such as permissions, which... we don't offer yet)

Reply:
> any associated relational data will be deleted

This ☝️ 

However, to err on the side of simplicity, I have removed the sentence entirely.